common/util/rhmds-api.js

```diff
@@ -171,11 +171,13 @@ const createPushApp = async name => {
   return pushApp;
 };
 
-const deployShowcaseServer = async (name, namespace) => {
-  await exec(`svcat provision ${name} \
-    --class datasync-showcase-server \
-    --plan default -n ${namespace} --wait \
-  `);
+const deployShowcaseServer = async (namespace) => {
+  await exec(`oc new-app --template datasync-showcase-server -n ${namespace}`);
+  await waitFor(async () => {
+    const replicasReady = (await exec(`oc get dc -o jsonpath='{.items[*].status.readyReplicas}' -n ${namespace} | wc -w`)).stdout.replace(/\s/g, '')
+    console.log(`Still waiting for deployment of Data Sync Template. Replicas ready: ${replicasReady}/3`)
```

Comment:
Maybe we can print a less verbose message
```suggestion
    console.log(`Waiting for showcase server: ${replicasReady}/3`)
```